tutorial.md

```diff
@@ -0,0 +1,68 @@
+## Installation
+
+Install via [git](https://git-scm.com/):
+
+```bash
+git clone https://github.com/eBay/sbom-scorecard
+```
```

Comment:
For users, we probably just want them to install the pre-compiled binaries. We're currently only generating them for linux, but you can find them here.

https://github.com/eBay/sbom-scorecard/releases/tag/0.0.3

So let's maybe tell them to just download the relevant version for their platform and we can work to add the right outputs for our next release?

Reply:
That makes sense to me. I'll update the PR.